bpfd/src/server/bpf.rs

```diff
@@ -149,7 +149,15 @@ impl<'a> BpfManager<'a> {
             }
             // Keep old_program until the dispatcher has been reloaded
             let _old_program = programs.remove(&id).unwrap();
+            info!(
+                "Program removed: {} programs attached to {}",
+                programs.len(),
+                &iface,
+            );
             if programs.is_empty() {
+                // Keep old_program until the dispatcher has been reloaded
+                let _old_programs = self.programs.remove(&iface).unwrap();
```

Comment:
The comment is incorrect - copy/pasted?
We don't need to keep `_old_programs` in scope as it's just an empty hashmap.
`self.programs.remove(&iface);` will work just fine.

Reply:
Done